server/providers/combined/tools/analyze-feature-scope/core-logic.ts

```diff
@@ -80,8 +80,8 @@ export async function executeAnalyzeFeatureScope(
     debugDir,
     figmaFileKey,
     yamlContent,
-    epicContext,
-    contentWithoutSection,
+    epicSansShellStoriesMarkdown: epicContext,
```

Comment:
Use `epicWithoutShellStoriesMarkdown`


`Without` is more comman than `Sans` outside design contexts